content/en/docs/faq.md

```diff
@@ -2,7 +2,7 @@
 title: "Frequently asked questions"
 description: "Frequently asked questions about Notary"
 type: docs
-weight: 8
+weight: 7
 ---
 
 ## What registries are compatible with Notary?
```

Comment:
```suggestion
## What registries are compatible with Notation?
```
Also need to make the same change in the next line "The following registries are compatible with ~~Notary~~ Notation for OCI artifact signing and verification: